org/istio.yaml

```diff
@@ -44,6 +44,7 @@ orgs:
     - clyang82
     - cmluciano
     - costinm
+    - craigbox
```

Comment:
hi @craigbox ! Welcome to the Istio community. This list is ASCII-betical. Can you correct the ordering?